### PR TITLE
nrnivmodl-core: adding -n name and dropping install script

### PR DESCRIPTION
**Pull request by ferdonline**
Install script was transformed into a config file being saved/loaded

### CODE REVIEW

**Comment by pramodk** [comment]
Please retest

File: extra/nrnivmodl-core.in [hunk 1]
```diff
@@ -28,8 +31,9 @@ while getopts "n:v:o:i:l:p:hV" OPT; do
     h)
         echo "$APP_NAME [options, ...] [mods_path]"
         echo "Options:"
-        echo "  -n <binary>     NMODL code generation compiler path"
-        echo "  -o <output_dir> Set the output dir (default: <arch>-core)"
+        echo "  -n <name>       The model name, used as a suffix in the shared library"
+        echo "  -m <mod2c_bin>  NMODL code generation compiler path"
+        echo "  -I <install_dir> Install to dir. Default: off - Only build in {ARCH}"
```
Comment (pramodk) [inline extra/nrnivmodl-core.in]: @ferdonline : nitpick : as we are using `-i` can we use different letter here? something like `-d` for destination or `-p` for path?

File: extra/nrnivmodl-core.in [hunk 2]
```diff
@@ -47,33 +52,39 @@ if [ $# -gt 1 ]; then
...
+echo "# nrnivmodl-core options. Mods location: $params_MODS_PATH" > $SAVE_FILE
+
 make_params=("ROOT=${ROOTDIR}")
 
 for param in $MAKE_OPTIONS; do
     var="params_${param}"
-    [ "${!var+x}" ] && make_params+=("$param=${!var}")
+    if [ "${!var+x}" ]; then
+        make_params+=("$param=${!var}")
+        echo "$var=${!var}" >> $SAVE_FILE
```
Comment (tristan0x) [inline extra/nrnivmodl-core.in]: The shell redirection can be set of the for level:

```
for ... ; do
 ...
done >> $SAVE_FILE
```

**Comment by pramodk** [comment]
@ferdonline : [Jenkins Pipeline](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/oss.coreneuron.pipeline/detail/oss.coreneuron.pipeline/687/pipeline):

```
[ 16%] Built target cudacoreneuron
[ 21%] Built target scopmath
[ 22%] Linking CXX static library ../lib/libcoreneuron.a
Running nrnivmodl-core with halfgap.mod
# nrnivmodl-core options. Mods location: /jenkins/06/workspace/oss.coreneuron.pipeline/tests/integration/ring_gap/mod
params_NMODL_BINARY=/jenkins/06/workspace/oss.coreneuron.pipeline/build_GPU-non-unified/bin/mod2c_core
params_INCFLAGS=-I/jenkins/06/workspace/oss.coreneuron.pipeline -I/jenkins/06/workspace/oss.coreneuron.pipeline/build_GPU-non-unified/include
/jenkins/06/workspace/oss.coreneuron.pipeline/build_GPU-non-unified/bin/nrnivmodl-core: line 60: -I/jenkins/06/workspace/oss.coreneuron.pipeline/build_GPU-non-unified/include: No such file or directory
make[2]: *** [lib/libcoreneuron.a] Error 1
make[2]: *** Deleting file `lib/libcoreneuron.a'
make[1]: *** [coreneuron/CMakeFiles/coreneuron.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by ferdonline** [comment]
retest this please

**Comment by ferdonline** [comment]
@ohm314 It seems I dont have right on this repo (?)

**Pull request merged**